og.module

```diff
@@ -44,6 +45,62 @@ function og_entity_insert(EntityInterface $entity) {
 }
 
 /**
+ * Implements hook_og_group_created().
+ */
+function og_og_group_created($entity_type, $bundle) {
```

Comment:
Why do we implement our own hook? Wouldn't it be better to handle this in `GroupManager::addGroup()`? This just adds unnecessary overhead of passing the data to a new function.